dialogue-apache-hc5-client/src/main/java/com/palantir/dialogue/hc5/InstrumentedManagedHttpClientConnection.java

```diff
@@ -95,7 +101,29 @@ public void sendRequestEntity(ClassicHttpRequest request) throws HttpException,
     @Override
     public ClassicHttpResponse receiveResponseHeader() throws HttpException, IOException {
```

Comment:
Looking for callers of this, it seems like it's only HttpRequestExecutor#execute that calls this, but as it's in a while loop, this could get a little trippy for status codes < 200 (i.e. 100 continue, 101 switching protocols)... i guess that's fine as those are presumably very rare?

Reply:
That's correct. We could limit this to the 2xx range, but even if we encounter 1xx responses they won't include the `Server-Timing` header.